web/src/page_params.ts

```diff
@@ -55,6 +57,7 @@ export const page_params: {
     show_plans: boolean;
     show_webathena: boolean;
     translation_data: Record<string, string>;
+    upload_space_used: number;
```

Comment:
I don't think this is correct -- `stats.ts` is a separate page and its `page_params` has a totally different type.

Reply:
OK I think the fix for this is something like to create a create a `web/src/stats/stats_page_params.ts` that is a copy of `page_params.ts` but with the appropriate types for that page's own parameters.

Reply:
Added `web/src/stats/page_params.ts` for `stat.js` as [`billing` ](https://github.com/zulip/zulip/pull/24559/commits/083d63eb51f90dac9299bff91e1a393a9e6d37ac) and [`portico`](https://github.com/zulip/zulip/pull/25864/commits/fc2808839a1a7ecfdaea2414904ccb687edb6480) modules did for their page_params.